libextism.pc

```diff
@@ -5,6 +5,6 @@ libdir=${exec_prefix}/lib
 
 Name: extism
 Description: The Extism universal plug-in system.
-Version: 0.0.1
+Version: 0.1.0
```

Comment:
Is this right? Should this reflect libextism's version?

Reply:
Yeah I think that's right